src/styles/main.scss

```diff
@@ -31,6 +40,14 @@ body {
   flex: 1;
 }
 
+:root {
+  --fontsize-h-1: 3.25rem; /* 52px */
+  --fontsize-h-2: 2.25rem; /* 36px */
+  --fontsize-h-3: 1.5rem; /* 24px */
+  --fontsize-h-4: 1.25rem; /* 20px */
+  --fontsize-h-5: 1.125rem; /* 18px */
+}
```

Comment:
Maybe you could at some point refactor these old HDS variables out and replace them with 
```--fontsize-heading-xl: 3.25rem;
--fontsize-heading-l: 2.25rem;
--fontsize-heading-m: 1.5rem;
--fontsize-heading-s: 1.25rem;
--fontsize-heading-xs: 1.125rem;
--fontsize-heading-xxs: 1rem;
```

Also for example `--fontsize-body-*` variables are renamed so please make sure that there are not old ones (e.g. `--fontsize-body-default`) in use

Reply:
Yeah I need to do that check. And I should do start using the new variables.. I will make a separate ticket and fix that soon

Reply:
tickets created